src/templates/kit/screens/TokensScreen.js

```diff
@@ -87,9 +84,37 @@ function Tokens({
     setTokenSymbol(event.target.value.trim().toUpperCase())
   }, [])
 
+  const membersRef = useRef()
+  const [focusLastMemberNext, setFocusLastMemberNext] = useState(false)
+
+  useEffect(() => {
+    if (!focusLastMemberNext || !membersRef.current) {
+      return
+    }
+
+    setFocusLastMemberNext(false)
+
+    // This could be managed in individual MemberField components, but using
+    // the container with a .member class makes it simpler to manage, since we
+    // want to focus in three cases:
+    //   - A new field is being added.
+    //   - A field is being removed.
+    //   - The first field is being emptied.
+    //
+    const elts = membersRef.current.querySelectorAll('.member')
+    if (elts.length > 0) {
+      elts[elts.length - 1].querySelector('input').focus()
+    }
+  }, [focusLastMemberNext])
+
+  const focusLastMember = useCallback(() => {
+    setFocusLastMemberNext(true)
+  }, [])
```

Comment:
I get an error that reads in the lines of "`useCallback` is not useful if no dependencies are set", is the compiler smart enough to not show it when there is a dependency that comes from a method that won't change?

Reply:
mmm that’s weird, `useCallback` is useful even with empty dependencies, to cache a function and use specific parameters like in this example. Was it an error coming from React itself?

Reply:
Merging for now as I can’t reproduce this error and it seems to be more of a warning displayed in development mode. Let me know if you find a way to reproduce it!